### PR TITLE
Mobile variant selectors

### PR DESCRIPTION
**Pull request by spacephoton**
x (A) Search bar stays on top (mobile) +cov
x (A) Horizontal scroll for mobile variant previews +cov
x (A) Metrics wrapper widen column threshold +cov
x (A) Move powered by vercel for mobile compatibility +cov
x (A) Variant selector on top +cov

### CODE REVIEW

**Comment by spacephoton** [comment]
CleanShot 2021-10-31 at 16 31 39

- Fixed controls positioning on mobile

**Comment by spacephoton** [comment]
CleanShot 2021-10-31 at 16 33 15
Reduce width of these metrics on top too

**Comment by spacephoton** [comment]
CleanShot 2021-10-31 at 16 42 55

**Comment by chaoran-chen** [comment]
Nice! The search on the top should make the mobile experience much better!

One issue: the width of the preview plots is too small. 
image

File: src/components/KnownVariantsList/KnownVariantsList.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useMemo, useState } from 'react';
-import styled from 'styled-components';
+// import styled from 'styled-components';
```
Comment (chaoran-chen) [inline src/components/KnownVariantsList/KnownVariantsList.tsx]: Please remove

**Review by chaoran-chen — APPROVED** [review]
I think that it would look better if the preview plots always have the same width. But it's up to you. There is one commented-out line of code (see below), otherwise, feel free to merge :)

**Comment by spacephoton** [comment]
> I think that it would look better if the preview plots always have the same width. But it's up to you. There is one commented-out line of code (see below), otherwise, feel free to merge :)

sounds good, set them to the same size

**Pull request merged**